common/src/Model/BrushFace.cpp

```diff
@@ -408,6 +412,39 @@ namespace TrenchBroom {
             m_texCoordSystem->shearTexture(m_boundary.normal, factors);
         }
 
+        void BrushFace::flipTexture(const vm::vec3& /* cameraUp */, const vm::vec3& cameraRight, const vm::direction cameraRelativeFlipDirection) {
```

Comment:
This looks relatively complex. I think a few tests would be good here.

Reply:
Yeah, agreed